2019/pt-br/src/0xb0-next-devs.md

```diff
@@ -0,0 +1,31 @@
+Próximos passos para Desenvolvedores
+====================================
+
+A tarefa de criar e manter software seguro, ou, corrigir software existente, pode ser uma tarefa difícil. APIs não são diferentes.
+
+Nós acreditamos que a educação e conscientização são fatores chave para escrever software seguro. Tudo o que é necessário para alcançar este objetivo depende de **estabelecer e usar repetidamente processos de  e padronizar controles de segurança**.
```

Comment:
fix
* where it's written "_(...) **estabelecer e usar repetidamente processos de  e padronizar controles de segurança**._"
* it should be "_(...) **estabelecer e usar procedimentos de segurança reprodutíveis e padronizar controles de segurança**._"

Reply:
okay